routes/api/v1/favorites.js

```diff
@@ -0,0 +1,75 @@
+var express = require("express");
+var router = express.Router();
+const fetch = require('node-fetch');
+var pry = require("pryjs");
+require("dotenv").config();
+var Location = require('../../../models').Location;
+var User = require('../../../models').User;
+var UserLocation = require('../../../models').UserLocation;
+
+router.post("/", function(req, res, next) {
+  if (req.body.api_key) {
+    User.findAll(
```

Comment:
Any reason why you decided to go with findAll instead of findOne? I see that you are then checking for the length of the array to catch unauthorized users, but then further down you have to do user[0], which feels weird to me. 

You could do User.findOne instead. findOne returns either the user or undefined, so you could do something like 
```
if(!user) 
```

instead of 
```
if (user.length === 0)
```

and then you get the user and don't have to use user[0]

Reply:
That's great feedback, thank you!